scripts/runBlackboxTests.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -eu
+# Currently the only test is whether oplogtoredis can connect
+# to a redis instance through TLS
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd $(dirname $SCRIPT_DIR)
+
+CERTS_DIR=$(pwd)"/blackbox-tests/certificates"
```

Comment:
you may want the start double-quote before `$(pwd)`. This is in case `pwd` has a space in it.